src/regions/ca/steps.json

```diff
@@ -362,6 +362,6 @@
   {
     "id": "outro2",
     "message": "outro2",
-    "trigger": ""
+    "end": true
```

Comment:
@vicrep I changed it to this - seems to work (it ends the flow) but not sure if that's what you meant?